be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -194,25 +194,36 @@ int64_t SinkBuffer::_network_time() {
 void SinkBuffer::cancel_one_sinker(RuntimeState* const state) {
     if (--_num_uncancelled_sinkers == 0) {
         _is_finishing = true;
-        if (_total_in_flight_rpc > 0) {
-            std::stringstream ss;
-            auto remain_rpc_num = 0;
-            for (auto& remain_rpc : _num_in_flight_rpcs) {
-                std::lock_guard<Mutex> l(*_mutexes[remain_rpc.first]);
-                if (remain_rpc.second > 0) {
-                    ss << (remain_rpc_num > 0 ? ", " : "") << print_id(_instance_id2finst_id[remain_rpc.first]);
-                    remain_rpc_num++;
+        if (state != nullptr && state->query_ctx()->is_query_expired()) {
```

Comment:
what if the query is not expired? we don't need to log?

Reply:
If the query is cancelled by users or other errors, the root causes may not from exchange. 
just if the query is blocked by exchange, we should know these msg.
if other cases need this logs, we can open it later.